bench/bench.ml

```diff
@@ -90,32 +90,85 @@ let prepare_workspace () =
       Format.eprintf "cloning %s/%s@." pkg.org pkg.name;
       Package.clone pkg)
 
-let dune_build () =
+let dune_build ?(suffix = "") () =
   let stdin_from = Process.(Io.null In) in
   let stdout_to = Process.Io.make_stdout Swallow in
   let stderr_to = Process.Io.make_stderr Swallow in
+  let gc_dump = Temp.create File ~prefix:"gc_stat" ~suffix in
   let open Fiber.O in
+  (* Build with timings and gc stats *)
   let+ times =
     Process.run_with_times dune ~display:Quiet ~stdin_from ~stdout_to ~stderr_to
-      [ "build"; "@install"; "--release" ]
+      [ "build"
+      ; "@install"
+      ; "--release"
+      ; "--dump-gc-stats"
+      ; Path.to_string gc_dump
+      ; "--sandbox"
```

Comment:
We should include sandboxing as a separate data point I think.

Reply:
OK, I'll do that in another PR to separate concerns.